src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala

```diff
@@ -129,7 +123,13 @@ object TpolecatPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     Def.derive(
-      scalacOptions ++= scalacOptionsFor(scalaVersion.value, tpolecatScalacOptions.value)
+      scalacOptions := {
```

Comment:
I have just noticed the change from `++=` back to `:=` - does that still behave correctly in relation to the issue you saw on #60?

Reply:
Yes. Look to back reference of `scalacOptions.value` in the body of the task

Reply:
I added a test to catch the problem. It should be working :)